test/modules/fundingManager/bondingCurveFundingManager/BancorVirtualSupplyBondingCurveFundingManager.t.sol

```diff
@@ -740,6 +740,7 @@ contract BancorVirtualSupplyBondingCurveFundingManagerTest is ModuleTest {
         uint feeAmount =
             (formulaReturn * fee) / bondingCurveFundingManager.call_BPS();
         uint sellAmountMinusFee = formulaReturn - feeAmount;
+        uint sellAmountPlusFee = feeAmount + sellAmountMinusFee;
```

Comment:
isnt this just the formulaReturn?

Reply:
Agree, just created a new var for readability in the test